sphinx/util/rst.py

```diff
@@ -103,6 +103,7 @@ def append_epilog(content: StringList, epilog: str) -> None:
     if epilog:
         if len(content) > 0:
             source, lineno = content.info(-1)
+            lineno = cast(int, lineno)  # lineno will never be None, since len(content) > 0
```

Comment:
Mmh. Ok (though I would have said "let's use an ignore then" since it's a comment rather than having a NO-OP function, but let's not spend time on those semantics).